README.md

```diff
@@ -54,4 +54,11 @@ To connect to AWS Keyspaces, the following environment variables need to be set:
 - `AWS_SECRET_ACCESS_KEY` - Your AWS Secret Access Key.
 - `AWS_SSL_CERTIFICATE_PATH` - The path to your SSL certificate for AWS Keyspaces.
 
-> 🗒️ **Note:** For convenience, an SSL certificate is provided in this repository and can be found at [/uptime_service_validation/database/aws_keyspaces/cert/sf-class2-root.crt](/uptime_service_validation/database/aws_keyspaces/cert/sf-class2-root.crt). Alternatively, the certificate can also be downloaded directly from AWS. Detailed instructions for obtaining the certificate are available in the AWS Keyspaces documentation, which you can access [here](https://docs.aws.amazon.com/keyspaces/latest/devguide/using_python_driver.html#using_python_driver.BeforeYouBegin).
\ No newline at end of file
+> 🗒️ **Note:** For convenience, an SSL certificate is provided in this repository and can be found at [/uptime_service_validation/database/aws_keyspaces/cert/sf-class2-root.crt](/uptime_service_validation/database/aws_keyspaces/cert/sf-class2-root.crt). Alternatively, the certificate can also be downloaded directly from AWS. Detailed instructions for obtaining the certificate are available in the AWS Keyspaces documentation, which you can access [here](https://docs.aws.amazon.com/keyspaces/latest/devguide/using_python_driver.html#using_python_driver.BeforeYouBegin).
+
+### Deployment
+
+The program needs:
+* The tables set up (see createDB.sql)
```

Comment:
Are all tables required there? Seeing `zk_validator` table there, but I think the data is now in Cassandra.

Reply:
Good spot. I thought I'd already deleted this, but I guess not.